node/libs/roles/src/validator/messages/consensus.rs

```diff
@@ -332,6 +283,21 @@ pub struct CommitQC {
 }
 
 impl CommitQC {
+    /// Create a new empty instance for a given `ReplicaCommit` message and a validator set size.
+    pub fn new(message: ReplicaCommit, validator_set_size: usize) -> Self {
+        Self {
+            message,
+            signers: Signers(BitVec::from_elem(validator_set_size, false)),
+            signature: validator::AggregateSignature::default(),
+        }
+    }
+
+    /// Add a validator's signature.
+    pub fn add(&mut self, sig: &Signature, validator_index: usize) {
```

Comment:
&ValidatorSet

Reply:
This one is the already-known index value, so I left as is.